src/Model/Config/Communication.php

```diff
@@ -123,4 +102,49 @@ protected function getChannel(string $langAbbr): string
 
         return $channels[$langAbbr];
     }
+
+    private function prepareParams(array $params, Category $category = null): array
+    {
+        $params['add-params'] = $this->useForCategories() ? $this->getStandardCategoryPath($category) : 'cl=search_result';
```

Comment:
This is incorrect and works different than before. the `cl=search_result` must be added to the add-params in search result page no no matter the FACT-Finder version is used
See the removed line 46